tools/docker-compose/README.md

```diff
@@ -118,6 +118,12 @@ $ make docker-compose
 
 > For running docker-compose detached mode, start the containers using the following command: `$ make docker-compose COMPOSE_UP_OPTS=-d`
 
+If you have encountered the infinitely-repeating `Waiting for postgres to be ready to accept connections` message during the execution, try to do the following:
```

Comment:
could you link some examples or commands for this section? I think can be helpful for novices.

Reply:
@masbahnana SGTM
- added a general example
- created an additional associated sub-section

thanks, PTAL